tools/mypy-onnx.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+import subprocess
+import os
+
+
+def main():
+    try:
+        subprocess.check_call(["mypy", "."])
+        subprocess.check_call(["mypy", "--py2", "."])
+
+        # Since test cases aren't a python package (missing __init__.py),
+        # mypy ignores them. Explicitly call mypy with these files.
+
+        root_folder = os.path.realpath(os.path.join(os.path.dirname(__file__), ".."))
```

Comment:
nit: Do `os.path.abspath` before doing `os.path.dirname` to avoid symlink confusion.
Use `os.pardir` instead of `..` , or you can just add another `os.path.dirname`.